front/types/files.ts

```diff
@@ -319,18 +319,18 @@ export const FILE_FORMATS = {
   "audio/mpeg": {
     cat: "audio",
     exts: [".mp3", ".mp4"],
-    isSafeToDisplay: false,
+    isSafeToDisplay: true,
```

Comment:
to be able to get the content of a file, it must be "safeToDisplay" otherwise the file controller forces the action `download`

we should have a `isSafeToDisplay` for the main file and its process version. 

But not today